compose: Change hover color on compose icons in dark theme.

jai2201 opened the pull request:
![Screencast-from-05-03-22-02_42_15-PM-IST](https://user-images.githubusercontent.com/76561593/156876929-7a3a9cbe-727d-4192-a956-7a7b0a17ef3c.gif)

jai2201 commented:
@timabbott , can you please review this PR, it changes the hover color on compose icons from blue to white to follow consistency.

CZO discussion -> https://chat.zulip.org/#narrow/stream/9-issues/topic/color.20on.20hover.20over.20compose.20icons.20in.20dark.20theme

alya commented:
Thanks for working on this!

I don't think white is the right solution for the hover color here. Our hover colors are pretty inconsistent, but probably the best color to match is the `+` and filter in the left sidebar. We should change the `^` and `x` in the compose box to this as well.

jai2201 commented:
Hmm, understood your concern over this ->

Posted a patch of comments suggesting alternatives to solve this bug ->
https://chat.zulip.org/#narrow/stream/9-issues/topic/color.20on.20hover.20over.20compose.20icons.20in.20dark.20theme

alya commented:
Please post an updated (still) screenshot + screen capture if you've had a chance to address the feedback. Thanks!

chdinesh1089 commented:
I think we should also try to make the [top right buttons' color](https://github.com/zulip/zulip/blob/main/static/styles/dark_theme.css#L526) consistent with these.

chdinesh1089 commented on static/styles/dark_theme.css:
```diff
@@ -400,6 +400,11 @@ body.dark-theme {
 
+    /* Override the color of icons with anchor tag of compose box on dark theme */
+    .compose_control_button:hover {
+        color: hsl(236, 31%, 90%);
```
Out of curiosity, where did we get these values from?

jai2201 commented:
Addressed these comments on CZO -> 
https://chat.zulip.org/#narrow/stream/9-issues/topic/color.20on.20hover.20over.20compose.20icons.20in.20dark.20theme/near/1341877

alya commented:
Thanks for your work on this @jai2201 ! Closing this Pr given our current plans to redesign the compose box.
